### PR TITLE
Capture buckets for data-pipeline

### PR DESCRIPTION
**Pull request by giorgiosironi**
/cc @de-code

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Created `data-pipeline--demo`:

```
$ aws s3 ls | grep demo
2018-08-03 11:58:27 demo-elife-data-pipeline
2018-08-03 11:58:26 demo-elife-data-pipeline-archive
```

File: projects/elife.yaml [hunk 1]
```diff
@@ -2018,3 +2018,13 @@ fastly-logs:
...
     aws-alt: {}
+
+data-pipeline:
+    description: Data consolidation project
+    domain: false
+    intdomain: false
+    aws:
+        ec2: false
+        s3:
+            "{instance}-elife-data-pipeline":
```
Comment (de-code) [inline projects/elife.yaml]: do we need a `deletion-policy`?

Comment (giorgiosironi) [reply projects/elife.yaml]: `delete` (throw away with the project instance) or `retain` (preserve after deletion)?

Comment (giorgiosironi) [reply projects/elife.yaml]: added

**Pull request merged**